perf/src/perf_libs.rs

```diff
@@ -20,5 +20,5 @@ pub fn append_to_ld_library_path(mut ld_library_path: String) {
         ld_library_path.push_str(&env_value);
     }
     info!("setting ld_library_path to: {ld_library_path:?}");
-    env::set_var("LD_LIBRARY_PATH", ld_library_path);
+    unsafe { env::set_var("LD_LIBRARY_PATH", ld_library_path) }
```

Comment:
I'm not sure how important this call is, but initialize poh is happening fairly late in the execution - I think to be sure we should move entry init poh earlier in https://github.com/anza-xyz/agave/blob/e9052f0e63bca0839b9b0eef8ca945c2d07eee0f/validator/src/commands/run/execute.rs#L114-L887

Reply:
Did that in https://github.com/anza-xyz/agave/pull/9501 as it's a change in startup sequence